Explorer++/Explorer++/RegistrySettings.cpp

```diff
@@ -145,6 +145,9 @@ LONG Explorerplusplus::SaveSettings(void)
 		NRegistrySettings::SaveDwordToRegistry(hSettingsKey,_T("LargeToolbarIcons"),m_bLargeToolbarIcons);
 		NRegistrySettings::SaveDwordToRegistry(hSettingsKey,_T("PlayNavigationSound"),m_bPlayNavigationSound);
 
+		NRegistrySettings::SaveDwordToRegistry(hSettingsKey,_T("DisplayMixedFilesAndFolders"),m_bDisplayMixedGlobal);
+		NRegistrySettings::SaveDwordToRegistry(hSettingsKey,_T("SortNamesInNonLogicalOrder"),m_bSortNonLogicalGlobal);

```

Comment:
1. No, the constants can just be regular C constants.

2. That sounds good to me. Though I'd probably just name it Settings.h.

3. Yeah, some parts aren't so great :smile: I wouldn't worry about it much for now.

For now, as long as the two new strings are in constants in RegistrySettings.cpp, that's good enough. Having the same string copied in two places could create issues down the line (if one string is updated and the other isn't, for example).

Or if you want to add a new header and put the constants in there, that works as well.